src/Filament/Resources/SiteResource.php

```diff
@@ -64,6 +64,30 @@ public static function form(Form $form): Form
                             ->when(SslCertificateCheckRepository::isEnabled())
                             ->label('SSL certificate check'),
                     ]),
+                Toggle::make('hardware_monitoring_notification_enabled')
+                    ->label('Alert Notification Enabled')
+                    ->inline(false),
+                Fieldset::make('Monitoring Limit')
+                    ->schema([
+                        TextInput::make('cpu_limit')
+                            ->label('CPU load is above (%)')
+                            ->numeric()
+                            ->default(80)
+                            ->minValue(1)
+                            ->maxValue(100),
+                        TextInput::make('ram_limit')
+                            ->label('Memory usage is above (%)')
+                            ->numeric()
+                            ->default(80)
+                            ->minValue(1)
+                            ->maxValue(100),
+                        TextInput::make('disk_limit')
+                            ->label('Disk usage is above (%)')
+                            ->numeric()
+                            ->default(80)
+                            ->minValue(1)
+                            ->maxValue(100),
+                    ]),
```

Comment:
Remember to group the TextInputs into 3 columns and the Toggle to cover the full width